yt/python/packages/build_ytsaurus_packages.sh

```diff
@@ -8,13 +8,15 @@ script_name=$0
 ytsaurus_source_path="."
 ytsaurus_build_path="."
 ytsaurus_package_name=""
+not_prepare_bindings_libraries=false
 
 print_usage() {
     cat << EOF
 Usage: $script_name [-h|--help]
                     [--ytsaurus-source-path /path/to/ytsaurus.repo (default: $ytsaurus_source_path)]
                     [--ytsaurus-build-path /path/to/ytsaurus.build (default: $ytsaurus_build_path)]
                     [--ytsaurus-package-name some-ytsaurus-package-name (default: all packages will be build) (values: ytsaurus-client, ytsaurus-yson, ytsaurus-local, ytsaurus-native-driver)]
+                    [--not-prepare-bindings-libraries]
```

Comment:
Maybe `binding-libraries`?

Reply:
In this case it will be inconsistent with this and other places
https://github.com/ytsaurus/ytsaurus/pull/379/files#diff-d7e1077c377a7d9234aeabf23282b2eb66f3c06286a1029e83636bdfc5078007R71

Also changing it in all places in this PR seems strange too